README.md

```diff
@@ -5,58 +5,47 @@
 [![GitHub stars](https://img.shields.io/github/stars/infinum/eightshift-boilerplate.svg?style=for-the-badge&label=Stars)](https://github.com/infinum/eightshift-boilerplate/)
 [![license](https://img.shields.io/github/license/infinum/eightshift-boilerplate.svg?style=for-the-badge)](https://github.com/infinum/eightshift-boilerplate)
 
-# Eightshift WordPress Boilerplate
+## About Eightshift Boilerplate
 
-This repository contains all the tools you need to start building a modern WordPress project, using all the latest front end development tools.
+Eightshift Boilerplate is a modern WordPress theme boilerplate. It includes everything you need to start building a modern WordPress project, using modern front end development tools, and it's at the core of the Eightshift Development Kit. [Find out more in Eightshift Docs &rarr;](https://infinum.github.io/eightshift-docs/)
```

Comment:
```suggestion
Eightshift Boilerplate is a modern WordPress theme boilerplate. It includes everything you need to start building a modern WordPress project, using state-of-the-art front-end development tools, and it's at the core of the Eightshift Development Kit. [Find out more in Eightshift Docs &rarr;](https://infinum.github.io/eightshift-docs/)
```

Or similar wording because we already have `modern WordPress` in the same sentence.